TCAT/Controllers/ActionOnboardViewController.swift

```diff
@@ -42,67 +51,141 @@ class ActionOnboardViewController: UIViewController, CLLocationManagerDelegate {
     func createView() {
         
         let spacing: CGFloat = 16
+        let edgeInset: CGFloat = 2
+        
+        let descriptionColor = UIColor(red: 153/255, green: 153/255, blue: 153/255, alpha: 1)
```

Comment:
try `.mediumGrayColor`? `mediumGrayColor` is  `UIColor(white: 155 / 255, alpha: 1.0)`